ui/src/app/components/workspace/workspace.component.scss

```diff
@@ -96,3 +101,7 @@ app-object-detail {
 ::ng-deep .column-right .mat-tab-header-pagination-after {
   margin-right: 40px;
 }
+.spanner-tab-link-workspace{
```

Comment:
can be moved to styless.cs to remove duplication

Reply:
Updated.